attributes/default.rb

```diff
@@ -69,7 +69,8 @@
   default['squid']['package'] = 'squid3'
   default['squid']['config_dir'] = '/etc/squid3'
   default['squid']['config_file'] = '/etc/squid3/squid.conf'
-  default['squid']['service_name'] = 'squid3'
+  default['squid']['service_name'] = 'squid3' if node['platform_version'].to_i <  16
```

Comment:
This should just be an if statement after a single default['squid']['service_name']